sdk/communication/AzureCommunicationCalling/Swift_Generated_Interfaces/ACSRendererView.h

```diff
@@ -0,0 +1,21 @@
+//+-----------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//
+// Module name : ACSRendererView.h
+//
+//------------------------------------------------------------------------------
+
+import Foundation
+import UIKit
+
+// Forward declaration as importing here causes cyclic dependency
+
+open class RendererView : UIView {
+
+    
+    open func update(scalingMode: ScalingMode)
+
+    open func dispose()
+
+    open func isRendering() -> Bool
+}
```

Comment:
Why is this necessary?  This will only work with UIKit. What is someone wants to use the Calling library with SwiftUI?

Reply:
We have been using this SDK with iOS application written in SwiftUI from day 1. We are not aware of any issues that the calling SDK does not work with SwiftUI.

Reply:
@tjprescott can you clarify more on this? maybe i am missing something.

Reply:
What's the link to your SwiftUI sample?

Reply:
I am curious to see / try this out. I think because it is a UIView and not a ViewController SwiftUI should be able to work.

Reply:
This is the link to our internal test-app sample written in SwiftUI is [here](https://skype.visualstudio.com/SCC/_git/client_calling_spool-native?path=%2Ftest_app%2FiOS%2FACSCallingTestApp%2FACSCallingTestApp%2FViews%2FCallInProgressView.swift&version=GBsanathr%2Fapi_review_2&line=89&lineEnd=94&lineStartColumn=1&lineEndColumn=22&lineStyle=plain&_a=contents).

Using Storyboard sample is [here](https://github.com/Azure-Samples/communication-services-ios-calling-hero-pr).

Reply:
Marking this as Resolved for now, please re-open if you have any more questions. Thanks.

Reply:
Swift UI sample in our repo is (here)[https://skype.visualstudio.com/SCC/_git/client_calling_spool-native?path=%2Ftest_app%2FiOS%2FACSCallingTestApp%2FACSCallingTestApp%2FViews%2FCallInProgressView.swift&version=GBmaster&line=729&lineEnd=730&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents]

Storyboard sample is (here)[https://github.com/Azure-Samples/communication-services-ios-calling-hero-pr/blob/main/ACSCall/Views/ParticipantView.swift]